urdfenvs/tiagoReacher/resources/tiagoRobot.py

```diff
@@ -11,14 +7,54 @@
 class TiagoRobot(DifferentialDriveRobot):
     def __init__(self):
         n = 19
-        self._r = 0.1
-        self._l = 0.4044
         fileName = os.path.join(os.path.dirname(__file__), "tiago_dual.urdf")
         super().__init__(n, fileName)
+        self._r = 0.1
+        self._l = 0.4044
 
     def setJointIndices(self):
-        self.robot_joints = [6, 8, 18, 19, 20, 22, 23, 24, 25, 26, 27, 28, 37, 38, 39, 40, 41, 42, 43]
-        self.urdf_joints = [6, 8, 19, 20, 21, 23, 24, 25, 26, 27, 28, 29, 38, 39, 40, 41, 42, 43, 44]
+        self.robot_joints = [
+            6,
+            8,
+            18,
+            19,
+            20,
+            22,
+            23,
+            24,
+            25,
+            26,
+            27,
+            28,
+            37,
+            38,
+            39,
+            40,
+            41,
+            42,
+            43,
+        ]
+        self.urdf_joints = [
+            6,
+            8,
+            19,
+            20,
+            21,
+            23,
+            24,
+            25,
+            26,
+            27,
+            28,
+            29,
+            38,
+            39,
+            40,
+            41,
+            42,
+            43,
+            44,
+        ]
         self.castor_joints = [9, 10, 11, 12, 13, 14, 15, 16]
         # TODO: This could be used as a starting point to get joint indices from urdf <01-12-21, mspahn> #
```

Comment:
you want this todo in the master branch? It could be a new issue.

Reply:
True! I will remove it and add a new issue for automated joint extraction.

Reply:
What other todo's do you mean?

Reply:
Todo moved to #57.

Reply:
Removed in https://github.com/maxspahn/gym_envs_urdf/pull/56/commits/2d8b76b0d4fdcfad20c44903abd4f226ae960484